### ISSUE TITLE
How to customize scroll behavior ? 

### ISSUE THREAD

**Issue by Kolyall** [rubensousa/RecyclerViewSnap#22]
```
 SnapHelper snapHelper = new GravitySnapHelper(Gravity.TOP);
  snapHelper.attachToRecyclerView(mRecyclerView);
```
For example I need to show next element if it is visible more than 50%. Or hide first element if it's hidden more than 50%.
Because current  behavior is :
if we scroll down and first item is hidden by 80%, then SnapHelper scrolls up the list and first element is visible again. 
Expected behavior : 
if we scroll down and first item is hidden by 80%, then SnapHelper should scroll down to hide first item.

**Comment by rubensousa** [comment]
I could add support for custom offsets. Currently, it collapses to an item if it's visible more than 50%.

**Comment by Kolyall** [comment]
Oh, than I found an issue. 50% collapsing doesnt work for grid layout manager

**Comment by Kolyall** [comment]
Because for linear it works properly

**Comment by Kolyall** [comment]
@rubensousa do you confirm the issue?
> 50% collapsing doesnt work for grid layout manager

**Comment by rubensousa** [comment]
@Kolyall yes, I do. Never thought someone would use this with a GridLayoutManager, so I didn't add support for it. I'll see what I can do.

**Comment by rubensousa** [comment]
Released on 1.2
